domains/certificates/Query.API/API.IntegrationTests/Factories/QueryApiWebApplicationFactory.cs

```diff
@@ -110,6 +117,18 @@ protected override IHost CreateHost(IHostBuilder builder)
         return client;
     }
 
+    public (WalletService.WalletServiceClient, Metadata metadata) CreateWalletClient(string subject)
+    {
+        var authentication = new AuthenticationHeaderValue("Bearer", GenerateToken(sub: subject));
```

Comment:
`GenerateToken` does it uses the Tokenvalidator library?

Reply:
GenerateToken is the function on line 134, which uses the TokenValidator library, yes